tests/covdefaults_test.py

```diff
@@ -1,5 +1,7 @@
 import importlib
+import os
 import re
+from pathlib import Path
```

Comment:
please don't use pathlib, thanks

Reply:
anything in special that makes you hate it? 😊